Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -0,0 +1,198 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+private enum PostCampaignCheckoutLayout {
+  enum Style {
+    static let cornerRadius: CGFloat = Styles.grid(2)
+  }
+}
+
+final class PostCampaignCheckoutViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var titleLabel = UILabel(frame: .zero)
+
+  private lazy var paymentMethodsViewController = {
+    PledgePaymentMethodsViewController.instantiate()
+    // TODO: Add self as delegate and add support for delegate methods.
+  }()
+
+  private lazy var pledgeCTAContainerView: PledgeViewCTAContainerView = {
+    PledgeViewCTAContainerView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+    // TODO: Add self as delegate and add support for delegate methods.
```

Comment:
I wasn't sure if we have a ticket for wiring up the payment flow here; let me know if we do and I'll link it